validation.js

```diff
@@ -0,0 +1,54 @@
+/* ---------------- contact form variables ----------------*/
+
+const form = document.querySelector('.contact-form');
+const emailRegex = /^([a-z|\d])+@+([a-z]){2,}\.+([a-z)]{2,})/;
+const emailInput = document.getElementById('email');
+const nameInput = document.getElementById('name');
+const msgInput = document.getElementById('message');
+const emailErrorMessage = document.querySelector('.email-error');
+const formErrorMessage = document.querySelector('.form-error');
+const nameErrorMessage = document.querySelector('.name-error');
+const messageErrorMessage = document.querySelector('.message-error');
+
+/* ---------- Contact Form -------------- */
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  if (nameInput.value !== null && msgInput.value !== null && emailRegex.test(emailInput.value)) {
+    form.submit();
+    formErrorMessage.textContent = 'Succesfully sent';
+    formErrorMessage.style.color = 'green';
```

Comment:
- [ ] As part of the requirements you are supposed to check if the email is in lower case and if not display an error message. kindly implement that too

Reply:
@m05e5  hey, thanks for your time. I would like to ask you something; I already checked it in line 17. I'm not sure what you want precisely. Can you explain more, please?

Reply:
@m05e5 actually I am checking all inputs

Reply:
@m05e5 when you try to enter invalid input in the email field so you will get an error.